HealthVault/Classes/Caching/Private/MHVThingCacheDatabase.m

```diff
@@ -702,16 +702,23 @@ - (NSError *)saveContext
     
     __block NSError *error = nil;
     
-    [self.managedObjectContext performBlockAndWait:^
-     {
-         if ([self.managedObjectContext hasChanges])
+    @try
```

Comment:
I accidentally deleted my real DB password when making some tests.  CoreData throws an exception here if the database is corrupted, catching and returning as error so that doesn't crash the app